DefaultTranslationPlugin.inc.php

```diff
@@ -3,86 +3,75 @@
 /**
  * @file DefaultTranslationPlugin.inc.php
  *
- * Copyright (c) 2013-2020 Simon Fraser University
- * Copyright (c) 2003-2020 John Willinsky
+ * Copyright (c) 2013-2022 Simon Fraser University
+ * Copyright (c) 2003-2022 John Willinsky
```

Comment:
I always forget to ask about these copyright dates, isn't it feasible to adopt a format like "Copyright (c) 2015 - Present XXXX"?

Reply:
To be honest I'm not sure, and have just kept updating the dates for consistency's sake. If you can find an example of a major project that does this, I'd be happy to adopt.

Reply:
Ok! I've created an issue with what I found here: https://github.com/pkp/pkp-lib/issues/7818